extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -199,7 +199,10 @@ class CornerstoneViewportService implements IViewportService {
       newViewportId = publicViewportOptions.viewportId;
       viewportInfo.setViewportId(newViewportId);
 
-      renderingEngine.disableElement(viewportId);
+      if (viewportId) {
+        console.log('Would disable element', viewportId);
+        // renderingEngine.disableElement(viewportId);
```

Comment:
why this is commented out? if the viewportId changes, we would want to disable the previous one. Example: when we switch hanging protocol e.g., MPR

Reply:
Changed